ingenialink/dictionary.py

```diff
@@ -268,6 +277,13 @@ def _read_xdf_register(self, register):
             dict: The current register which it has been reading
             None: When at least a mandatory attribute is not in a xdf file
 
+        Raises:
+            KeyError: If the register doesn't have an identifier.
+            ILValueError: If the register data type is invalid.
+            ILAccessError: If the register access type is invalid.
+            ILUnsupportedRegisterValue: If the register address type is invalid.
```

Comment:
We use IL exceptions when the exceptions are related to a drive error. Here you can simply use ValueError or another standard python exception.